runtime/caches/s3.ts

```diff
@@ -29,6 +30,7 @@ const awsRegion = Deno.env.get("CACHE_AWS_REGION");
 const awsAccessKeyId = Deno.env.get("CACHE_AWS_ACCESS_KEY_ID")!;
 const awsSecretAccessKey = Deno.env.get("CACHE_AWS_SECRET_ACCESS_KEY")!;
 const awsEndpoint = Deno.env.get("CACHE_AWS_ENDPOINT");
+const awsFolder = Deno.env.get("DECO_SITE_NAME") ?? "general_cache_deco";
```

Comment:
Interesting. But we have multi-tenant sites. We should use `Context.active().site` instead.